recipes/basemap-data-hires/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+    name: basemap-data-hires
+    version: 1.0.8.dev
+
+source:
+  git_url: https://github.com/matplotlib/basemap.git
+  git_tag: 116aefb
```

Comment:
Basemap `1.0.7` does not compile with geos > 3.3.3 on Windows but latest master does. It is not good practice, but basemap's releases are very slow to wait for a new one.